features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/default/js/edit.asset.js

```diff
@@ -86,7 +86,7 @@ $(function() {
 			},
 			success : function(response) {
 
-				var result = JSON.parse(response);
+				var result = response;
```

Comment:
removed redundant variable 'result' in commit 3ff3759d84c0361d76bba01f7c69eafb4878291e of PR #240